### PR TITLE
Support STP parameters in bridge

### PR DESCRIPTION
**Pull request by daniel-noland**
Added basic support for some basic spanning tree protocol configuration
parameters on the bridge device.

### CODE REVIEW

**Comment by daniel-noland** [comment]
Notes:

1. I am very new to Go, so if I am deviating from any idioms or best practices please let me know and I will do my best to correct.
2. When I run the full `go-netlink` test suite some of the tests fail due to parameter off-by-one issues.  I am running linux kernel 5.11.10 at the time of writing.  I don't know why this is happening for me but I will assume this is not happening elsewhere until I dig in a little more and find out what is going on.  I mention this here because I deliberately wrote a couple more tests which fail for similar off-by-one issues under the assumption that they won't fail on other machines.  I am leaving this as a draft PR till I find out what is going on.

**Comment by daniel-noland** [comment]
*Warning:*

My colleague ran the tests I included in this patch on his machine (running linux kernel 5.4) and all tests pass.

I am upgrading this PR from draft in the hopes that others will run the tests included in this patch and confirm.

I suggest that we make a point of running this test suite on various linux kernels.

*The tests fail for me on kernel 5.11.10*

The failure symptom is that all timer values I specify are accepted by the kernel (assuming they were otherwise valid).  However, on query every timer is reported as the specified value less one.  E.g., if I specify the max age stp parameter as 200 then the operation is is accepted but on querying the current value it returns as 199.

File: link_linux.go [hunk 1]
```diff
@@ -303,6 +303,46 @@ func (h *Handle) BridgeSetVlanFiltering(link Link, on bool) error {
...
+	return pkgHandle.BridgeSetVlanProtocol(link, protocol)
+}
+
+func (h *Handle) BridgeSetVlanProtocol(link Link, protocol string) error {
+	bridge := link.(*Bridge)
+	bridge.VlanProtocol = &protocol
+	return h.linkModify(bridge, unix.NLM_F_ACK)
+}
+
+func (h *Handle) BridgeSetStpState(link Link, state uint32) error {
```
Comment (aboch) [inline link_linux.go:316]: What values can _state_ take?

**Comment by aboch** [comment]
also pls push again so CI is triggered

**Comment by aboch** [comment]
ping...

**Comment by vishvananda** [comment]
Looks like the tests are failing in CI here

**Comment by aboch** [comment]
@daniel-noland

**Comment by aboch** [comment]
@daniel-noland

**Comment by vishvananda** [comment]
Closing due to inactivity. Feel free to reopen.
